lib/safira_web/live/backoffice/event_live/index.ex

```diff
@@ -58,4 +59,35 @@ defmodule SafiraWeb.Backoffice.EventLive.Index do
     |> assign(:page_title, "New FAQ")
     |> assign(:faq, %Faq{})
   end
+
+  defp apply_action(socket, :teams, _params) do
+    socket
+    |> assign(:page_title, "Teams")
+  end
+
+  defp apply_action(socket, :teams_new, _params) do
+    socket
+    |> assign(:page_title, "New Team")
+  end
+
+  defp apply_action(socket, :teams_edit, %{"team_id" => team_id}) do
+    socket
+    |> assign(:page_title, "Edit Team")
+    |> assign(:team, Teams.get_team!(team_id))
+  end
+
+  defp apply_action(socket, :teams_members, %{"team_id" => team_id}) do
+    socket
+    |> assign(:page_title, "Team Members")
+    |> assign(:team, Teams.get_team!(team_id))
+    |> assign(:members, Teams.list_team_members(team_id))
+    |> assign(:member, %Teams.TeamMember{})
+  end
+
+  defp apply_action(socket, :teams_members_edit, %{"id" => member_id}) do
+    socket
+    |> assign(:page_title, "Edit Team Member")
+    |> assign(:team, Teams.get_team_member!(member_id).team)
+    |> assign(:member, Teams.get_team_member!(member_id))
```

Comment:
No need to repeat this twice, just fetch the team member, and add an error handling when it doesnt exist.